ansible/roles/test/files/ptftests/vnet_vxlan.py

```diff
@@ -0,0 +1,277 @@
+# ptf -t "config_file='/tmp/vnet_vxlan.json';vxlan_enabled=True" --platform-dir ptftests --test-dir ptftests --platform remote vnet-vxlan
+
+# The test checks vxlan encapsulation/decapsulation for the dataplane.
+# The test runs three tests for each vlan on the DUT:
+# 1. 'FromVM'   : Sends encapsulated packets to PortChannel interfaces and expects to see the decapsulated inner packets on the vlan interface.
+# 2. 'FromServ' : Sends regular packets to Vlan member interface and expects to see the encapsulated packets on the corresponding PortChannel interface.
+# 3. 'Serv2Serv': Sends regular packets to Vlan member interfaces and expects to see the regular packets on the one of Vlan interfaces.
+#
+# The test has the following parameters:
+# 1. 'config_file' is a filename of a file which contains all necessary information to run the test. The file is populated by ansible. This parameter is mandatory.
+
+import sys
+import os.path
+import json
+import ptf
+import ptf.packet as scapy
+from ptf.base_tests import BaseTest
+from ptf import config
+import ptf.testutils as testutils
+from ptf.testutils import *
+from ptf.dataplane import match_exp_pkt
+from ptf.mask import Mask
+import datetime
+import subprocess
+from pprint import pprint
+
+class VNET(BaseTest):
```

Comment:
I understand the config is static in the test. set before the tests starts and remains unchanged until the end od the test. How about adding dynamicity to the test. that is, adding /removing routes during the test?

Reply:
I agree that adding some state transition coverage would make the test cases stronger.